internal/compiler/passes/remove_aliases.rs

```diff
@@ -190,18 +197,13 @@ fn is_declaration(x: &NamedReference) -> bool {
 }
 
 /// Out of two named reference, return the one which is the best to keep.
-fn best_property(
-    component: &Rc<Component>,
-    p1: NamedReference,
-    p2: NamedReference,
-) -> NamedReference {
+fn best_property(p1: NamedReference, p2: NamedReference) -> NamedReference {
     // Try to find which is the more canonical property
     macro_rules! canonical_order {
         ($x: expr) => {{
             (
                 !$x.element().borrow().enclosing_component.upgrade().unwrap().is_global(),
                 is_declaration(&$x),
-                !Rc::ptr_eq(&component.root_element, &$x.element()),
```

Comment:
Ah, this would previously prioritise the properties declared in the root element over others? I guess with the move of everything to the root this isn't needed anymore, even if we still had the component parameter?

Reply:
Right, this was never really needed anyway,  just a preference i had.